build.gradle

```diff
@@ -236,7 +236,9 @@ project('client') {
         compile project(':shared:authplugin')
         compile project(':shared:protocol')
         compile project(":shared:controller-api")
-        compile group: 'io.netty', name: 'netty-transport-native-epoll', version: nettyVersion
+        // Ensure epoll native transport is used for linux-x86_64 architecture systems.
+        // The client fallsback to NIO based transport if epoll is not available.
+        compile group: 'io.netty', name: 'netty-transport-native-epoll', version: nettyVersion, classifier: 'linux-x86_64'
```

Comment:
FYI this should be removed from here since we have a dedicated PR for it (#4262 ).